_pages/security/static-analysis.md

```diff
@@ -16,26 +16,32 @@ There are tools for JS, Ruby, and Python, and you are encouraged to set up this
 Service | Languages/frameworks | Dependency analysis | Code security analysis
 --- | --- | --- | ---
 [Code Climate](https://codeclimate.com/) | Ruby, JS, Python | Ruby and Node (on `push` only) | Rails, Node, and Ember
-[Gemnasium](https://gemnasium.com/) | Ruby, Node, Python, PHP | Y | N
+[Gemnasium](#gemnasium) | Ruby, Node, Python, PHP | Y | N
 [Hakiri](https://hakiri.io/) | Ruby | Y | Rails only
 
 Code analysis can be run locally with the following open source tools. These tools provide results similar (and in some cases, identical) to the hosted services above.
 
-#### Gemnasium setup
+#### Gemnasium
 
-To set up Gemnasium, you'll need to be invited to the devops@gsa.gov account (ask in [#infrastructure](https://gsa-tts.slack.com/messages/infrastructure/)). In this process, be sure to log in via a password (rather than Github login) as that'll tie access to your gsa.gov email (as opposed to personal email).
+##### Setting up your account
+
+1. [Sign into Gemnasium via GitHub.](https://gemnasium.com/login)
+1. Go to [your settings page](https://gemnasium.com/settings) and change it to use your work email, if it isn't already.
```

Comment:
There are currently several accounts using personal email addresses. I'll reach out to those folks and ask them to update. Everything else looks fine. The discussion about GH accounts was well before my time here and I'm not seeing any references to it.